package/yast2-ntp-client.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-ntp-client
-Version:        3.2.1
+Version:        3.2.2
```

Comment:
that reminds me, one important thing is missing here. Dependency on cfa gem and also with correct version that have your changes.